helper/database.js

```diff
@@ -0,0 +1,24 @@
+const blueBird = require('bluebird');
+const dbClient = require('mongodb').MongoClient;
+const config = require('../config');
+const databaseSettings = config.databaseSettings;
+
+let initializeDatabase = () => {
+    let uriString = databaseSettings.getDatabaseUrl(databaseSettings.getDatabaseConfig());
+    let connectionString = 'mongodb://' + uriString;
+
+    return dbClient.connect(connectionString, {
+        promiseLibrary: blueBird
+    })
+        .then((dbClient) => {
+            console.log("Successfully connected with database!");
+            module.exports.dbClient = dbClient;
+        })
+        .catch((e) => {
```

Comment:
This will catch the error without killing the application. Is that intended? Should catch and log, and reject/throw the error so the callee can exit the application. So in www.js, catch the database.connect-function and use process.exit(1) to exit the application if it fails to connect to the database. If we don't do this, we should have some sort of reconnect-functionality in place  :)